scripts/sync_org_skills.py

```
@@ -0,0 +1,68 @@
+    "contributor-onboarding",
+    "project-template",
+    "mcp-integration",
+    "GLOSSARY.md",
+]
+
+SKILLS_REPO_ZIP = "https://github.com/AOSSIE-Org/Skills/archive/refs/heads/main.zip"
+TARGET_DIR = repo_root / "org-wide-skills"
+
+
+def sync_org_skills():
+    logger.info(f"Downloading org skills archive from {SKILLS_REPO_ZIP}...")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)